media-sound/pure-data/pure-data-0.47_p1.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools eutils fdo-mime
+
+DESCRIPTION="Visual programming language for multimedia"
+HOMEPAGE="http://msp.ucsd.edu/software.html"
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/pure-data/${PN}.git"
+	KEYWORDS=""
+else
+	MY_PN="pd"
+	MY_PV=${PV/_p/-}
+	MY_P="${MY_PN}-${MY_PV}"
+	SRC_URI="http://msp.ucsd.edu/Software/${MY_P}.src.tar.gz
+					 https://puredata.info/portal_css/Plone%20Default/logo.png -> ${PN}.png"
```

Comment:
Yeah, that's because ebuilds have to be indented with tabs, so it depends on the editor/viewer's settings for tab-width.
Maybe it also works without the extra indentation, let me try.